hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -1305,7 +1305,6 @@ def _dtype_from_args(args):
     else:
         # Two args: ndarray[shape, type], NDArray[*]
         assert len(args) == 2
-        assert args[0] is Any
         dtype = _unpack_dtype(args[1])
```

Comment:
Just checking, we still expect the dtype type to be the second element here?

Reply:
yup, though I had to go find the pr numpy-side to convince myself of this: https://github.com/numpy/numpy/pull/27211